packages/react-native-nitro-modules/src/turbomodule/NativeNitroModules.ts

```diff
@@ -42,7 +42,7 @@ declare global {
 }
 
 export function isRuntimeAlive() {
-  const cache = global.__nitroModulesJSICache
-  const dispatcher = global.__nitroDispatcher
```

Comment:
Why? I don't think this makes sense.

Reply:
Using the `@tsconfig/react-native` config enforces to use `globalThis`. `globalThis` can be used as a drop-in replacement for `global``
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis

Reply:
<img width="793" alt="Screenshot 2024-11-04 at 13 09 02" src="https://github.com/user-attachments/assets/45029b3a-59c0-45a6-8d77-342c069aaa4b">

Reply:
sounds good, nice work 👍